### PR TITLE
Add step to load shasum in windows publish

### PR DESCRIPTION
**Pull request by koln67**
Load the shasum variable

### CODE REVIEW

**Comment by koln67** [comment]
/hold

only merge if the manual concourse testing shows that this steps is necessary for the windows publish to succeed

File: concourse/pipelines/windows-image-build.jsonnet [hunk 1]
```diff
@@ -819,6 +843,10 @@ local ImgGroup(name, images, environments) = {
              [
                common.GcsImgResource(image, 'windows-install-media')
                for image in windows_install_media_images
+             ] +
+             [
+               common.GcsShasumResource(image, 'windows-install-media')
```
Comment (jjerger) [inline concourse/pipelines/windows-image-build.jsonnet:848]: Did you intend to make this the shasum?

Comment (koln67) [reply concourse/pipelines/windows-image-build.jsonnet]: I added it so it passes the presubmit, however it doesn't have any function.

The windows media images don't use the sha-sum or sbom, however it uses the "imgpublishjob" which is common for all the windows builds.

Comment (koln67) [reply concourse/pipelines/windows-image-build.jsonnet]: Fixed: the current way skips the shasum steps for windows media images by passing in "generate_shasum = false"

**Review by jjerger — COMMENTED** [review]
/lgtm

**Pull request merged**